Workbooks/IntsightsIOCWorkbook.json

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
It is not working for me in a workspace I tried it in, the fiels "ReportingFeeds" does not exist.


![image](https://user-images.githubusercontent.com/38046493/99357926-74e09900-28b5-11eb-910c-7e628001755a.png)

Reply:
mmm yes you are right, in case that you don't have any reporting_feeds fields you will get an error, I can found a solution for that (like a default value to 'All' = [])
basically, the only problem is that we see an error instead of 'no data to show'
if there is any way to fix that please let me know, but I don't think it's a problem because there is no data to show anyway

the ReportingFeed field exists only if you will add a new 'Threat intelligence - TAXII' connection with our URL, you can use that to see how the workbook should be look like:

url: https://devapi.intsights.com/tip/integrations/taxii2/5fb405d635869500073c525a
collection id: 5fb405fd4c6c8e000738e16c
username: 5d70fcf999293c0007fb427d
password: d08da09cedf09f152f029fe041714b0208269de1bc7a1f6cb789edabbabce83c5b752539e5a5f1b8d19b26a24110ba5bb9166a6d1c8527d991eee23499f49960

 Basically, it should only be used by our customers - Intsights company

Reply:
What you can do is:
1. Making the parameter reporting feeds required:
![image](https://user-images.githubusercontent.com/38046493/99498371-04eb1500-2980-11eb-95d4-4beadf876e89.png)

2. Add a default value to all, for example - All

![image](https://user-images.githubusercontent.com/38046493/99498704-8d69b580-2980-11eb-891f-05e06bb4acb1.png)

Then it will look like this:
![image](https://user-images.githubusercontent.com/38046493/99498749-9d819500-2980-11eb-9421-4e70e6d9d841.png)

Reply:
It is still not perfect but I think it is good enough, the goal is that customers who do not have this field will have an empty experience and not broken experience.

Reply:
1. I updated the reporting_feeds to be required
2. About the 'Select All value' - it kind of weird if 'All' filter will be select nothing, instead of select all.
it's not a default value, this is the value he receives in case that we have at least one option or nothing